apps/design-land/src/app/media-gallery/media-gallery.component.html

```diff
@@ -0,0 +1 @@
+<design-land-example-viewer-container example="basic-media-gallery"></design-land-example-viewer-container>
```

Comment:
@xelaint needs docs.

Reply:
wrong PR?

Reply:
Working on docs right now. I think it should be added in the same PR as the creation of the component?

Reply:
Oh, I thought he meant docs for how to write example components for design-land lol. I found it a little confusing to be honest. Agreed that media-gallery docs go in this PR

Reply:
Yeah, I unfortunately replaced brevity for clarity, I did indeed mean docs for the media gallery.

Reply:
Looking back, it seems kind of obvious you meant the media gallery. Not sure why I took it otherwise